### PR TITLE
feat: add artisan command to create new account

### LINKED ISSUE #4744: Create User command for artisan
**Is your feature request related to a problem? Please describe.**
I installed an instance of Monica and want to add additional accounts for other family members, but I don't want to open my instance to any person. While the SetupProduction command allows you to create an initial account, there is no easy way to create additional accounts. (See https://github.com/monicahq/monica/issues/4617 for another example of this)

**Describe the solution you'd like**
I would like an artisan command to create a new account easily from the command line.

**Describe alternatives you've considered (optional)**
https://github.com/monicahq/monica/issues/1876 talks about an admin portal, but that's a ways off. We could use the API, but that is rather complicated for an occasional new user.

### PR DESCRIPTION
**Pull request by lpdyck**
This adds an Account:Create command to artisan so that accounts can be created on an instance without opening public registration.

Resolves #4744 

### Before submitting the PR
- [x] Read the CONTRIBUTING document before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Create your PR as draft if it is not final yet. Mark it as ready... when it’s ready. Otherwise the PR will be considered complete and rejected if it's not working.

### General checks
- [x] Make sure that the change you propose is the smallest possible.
- [x] The name of the PR should follow the [conventional commits guideline](https://github.com/monicahq/monica/blob/master/docs/contribute/readme.md#conventional-commits) that the project follows.

#### Other tasks
- [x] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG.md) entry added, if necessary, under `UNRELEASED`.
- [x] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary.
- [x] If it's relevant and worth mentioning, create a changelog entry for this change. The changelog entry will appear inside the UI for all users to see. To know if your change is worth the creation of a changelog entry, [read the documentation](https://github.com/monicahq/monica/blob/master/docs/administrators/tips.md#when-is-it-relevant-to-create-a-changelog-entry).
- [ ] Don't forget to [ask for a free account](mailto:regis@monicahq.com) on https://monicahq.com as anyone who contributes can request a free account.

### CODE REVIEW

File: app/Console/Commands/CreateAccount.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        $password = $this->option('password');
+        if (empty($password)) {
+            $this->error('! You must specify a password');
+        }
+
+        if (empty($email) || empty($password)) {
+            return;
+        }
+
+        Account::createDefault('John', 'Doe', $email, $password);
```
Comment (asbiin) [inline app/Console/Commands/CreateAccount.php:47]: I would rather add real firstname/lastname than a default "John Doe" here.
They could be optional input.

File: app/Console/Commands/CreateAccount.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Account\Account;
+use Illuminate\Console\Command;
+
+class CreateAccount extends Command
```
Comment (asbiin) [inline app/Console/Commands/CreateAccount.php:8]: Please add the `ConfirmableTrait` trait and a test like `$this->confirmToProceed()` (see [MigrateDatabaseCollation.php](https://github.com/monicahq/monica/blob/master/app/Console/Commands/MigrateDatabaseCollation.php) for instance)

**Review by asbiin — CHANGES_REQUESTED** [review]
Good idea! Thank you.
See my review

File: app/Console/Commands/CreateAccount.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+namespace App\Console\Commands;
+
+use App\Models\Account\Account;
+use Illuminate\Console\Command;
+use Illuminate\Console\ConfirmableTrait;
+
+class CreateAccount extends Command
+{
+    use ConfirmableTrait;
+    /**
```
Comment (asbiin) [inline app/Console/Commands/CreateAccount.php:12]:
```suggestion
    use ConfirmableTrait;
 
    /**
```

**Review by asbiin — APPROVED** [review]
Great

**Comment by djaiss** [comment]
Good, thank you!

**Comment by lpdyck** [comment]
I'm fairly certain the suddenly failing tests are coincidental to my adding a new test, but maybe I'm wrong?

**Comment by asbiin** [comment]
@lpdyck no it's not. I'm trying to resolve it in #4763

**Comment by asbiin** [comment]
Thank you @lpdyck !

**Pull request merged**